enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -21,67 +21,69 @@
         {# Display success, error, warning or info messages #}
         {% alert_messages messages %}
 
-        {% if course_enrollable %}
-          <h2 class="course-confirmation-title">{{ confirmation_text }}</h2>
-        {% endif %}
+        {% if course_and_course_run_available %}
```

Comment:
We are beginning to add a lot of conditional logic to this template. This seems like a good opportunity to clean up both the view and the template by rendering rendering a separate template for the error conditions.